strapi/strapi
# [helper-plugin]: migrate use-persistent-state hook to typescript

### What does it do?

Migrate helper-plugin/hooks/usePersistentState to typescript.
Added test case.

### How to test it?

You can check if any value in the local storage getting set properly or not, for eg: key: navbar-condensed(i.e. dashboard left menu)

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/17690
